game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -54,6 +66,18 @@ class GameThread(scoreThread: ScoreThread, playerIDs: List[Int]) extends Runnabl
   }
 
   def updateGames: Unit = {
+    // Test to see if the ranked is over
+    games.find(_.id == Game.RankedGameId) match {
+      case Some(ranked) => {
+        val difference = GameThread.time - ranked.startTime
```

Comment:
I'd personally prefer a name like `duration` or `elapsed` or similar, but we can keep `difference` if you think that's better.